pkg/controllers/state/cluster.go

```diff
@@ -362,6 +362,7 @@ func (c *Cluster) newStateFromMachine(machine *v1alpha5.Machine, oldNode *StateN
 		Machine:           machine,
 		hostPortUsage:     oldNode.hostPortUsage,
 		volumeUsage:       oldNode.volumeUsage,
+		volumeLimits:      oldNode.volumeLimits,
```

Comment:
Can we make sure that we copy over the inflight details as well? There's a few details here that weren't correctly mapped over.